README.md

```diff
@@ -1,25 +1,39 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+#  Math-Magicians React App
 
-## Available Scripts
+> "Math-Magicians"  is a webapp for all fans of mathematics. It is a Single-Page-App (SPA) that allows users to make simple calcualtions and read a randoom amth-related quote. This Single-Page-app was created using React
+
+
+## Built With
+
+- React 
+- CSS
+- Webpack and Babel
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
```

Comment:
- [ ] I would recommend removing the `live demo` section if you don't have a live demo link for now.